holidays/countries/united_states.py

```diff
@@ -186,6 +186,7 @@ def _add_subdiv_holidays(self):
         if self._year >= 1937 and self.subdiv not in {
             "AK",
             "AR",
+            "CA",
```

Comment:
I read it as like the Columbus day holiday never existed. As far as I remember it was just renamed somewhere around 2019th. Please confirm with refs to some sources if possible.

Reply:
They were removed altogether in 2009 for `CA` and `DE`.

`DC`, `ME`, `NM`, and `VT` on the other hand kept it with a name change to "Indigenous Peoples' Day" in 2019.

Though if we're at it, may as well go deep down and check other cases as well like Hawaii's names change to "Discoverers Day" sometime prior to 2000 and other instances just to be sure. 👀

Reply:
> They were removed altogether in 2009 for `CA` and `DE`.

Any sources?

Also https://www.gov.ca.gov/2019/10/14/governor-newsom-issues-proclamation-declaring-indigenous-peoples-day/

Reply:
https://www.pewresearch.org/short-reads/2023/10/05/working-on-columbus-day-or-indigenous-peoples-day-it-depends-on-where-your-job-is/

AFAIK California's Indigenous Peoples' Day declared in 2019 is an observed-only (no public holidays are given) https://en.m.wikipedia.org/wiki/Indigenous_Peoples%27_Day_(United_States)

Reply:
you're right it should be only removed since 2009 not before so should update the PR for that. Here are some news article sources at least and the state holiday calendar. Native American Day on the fourth friday of september was added in lieu of it but also seems to be an optional day off that only some state institutions observe.

https://www.latimes.com/nation/nationnow/la-na-nn-columbus-day-why-do-we-still-celebrate-20151012-htmlstory.html

> (Although California Gov. Arnold Schwarzenegger eliminated the Columbus Day state holiday as part of a budget-cutting measure in 2009, Los Angeles city and county offices still observe it. The Los Angeles Unified School District does not.)

https://www.pewresearch.org/short-reads/2023/10/05/working-on-columbus-day-or-indigenous-peoples-day-it-depends-on-where-your-job-is/

> California, for example, is among the 26 states that don’t have an official public holiday on the second Monday in October. But in the past, it has designated the day as [Indigenous Peoples’ Day by gubernatorial proclamation](https://www.gov.ca.gov/2022/10/10/governor-newsom-proclaims-indigenous-peoples-day-2022/). California law also designates the fourth Friday in September as Native American Day, which state employees may take in lieu of their annual personal holiday.

And here for the current year and previous you can see it's not on the state calendar: https://www.calhr.ca.gov/employees/pages/state-holidays.aspx